hash/msx1_cart.xml

```diff
@@ -5901,6 +5981,19 @@ kept for now until finding out what those bytes affect...
 		</part>
 	</software>
 
+	<software name="gomokunak" cloneof="gomokuna">
+		<description>Dunoegyosil Omog (Korea)</description>
+		<year>1984</year>
+		<publisher>Zemina</publisher>
+		<info name="alt_title" value="두뇌교실 오목" />
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x4000">
+				<rom name="dunoegyosil omog (korea).rom" size="0x4000" crc="f4cb0a83" sha1="bd5617cfd9ee1d17f4cde6f282da897003226089" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Correct RR for this title would be “Dunoegyosil Omok” (“ㄱ” is transliterated as “k” in final position where it’s pronounced as an unreleased stop).

Reply:
Thanks, fixed.